docs/compilers/CSharp/Compiler Breaking Changes - DotNet 7.md

```diff
@@ -427,3 +427,16 @@ class @required {} // No error
 This was done as `required` is now a member modifier for properties and fields.
 
 You can learn more about this change in the associated [csharplang issue](https://github.com/dotnet/csharplang/issues/3630).
+
+## Types cannot be named `file`
```

Comment:
Please place the new entry at the top. We changed the order when mirroring process to docs.microsoft.com started (was done in [https://github.com/dotnet/roslyn/pull/60260](https://github.com/dotnet/roslyn/pull/60260))